### PR TITLE
Minor changes to _episodes/03-create.md

### PR DESCRIPTION
**Pull request by rgaiacs**
Mainly making use of `<kbd>`.

### CODE REVIEW

File: _episodes/03-create.md [hunk 1]
```diff
@@ -447,18 +455,30 @@ quotes.txt
...
 > fructose.dat glucose.dat maltose.dat sucrose.dat
+> ~~~
+> {: .output}
+> ~~~
 > $ cd raw/
 > ~~~
 > {: .language-bash}
 >
-> Fill in the blanks to move these files to the current folder
+> Replace `placeholder1`, `placeholder2` and `placeholder3`PLACEHOLDER2 in the blanks
```
Comment (gcapes) [inline _episodes/03-create.md]: I think PLACEHOLDER2 is a typo?
Also I don't think the rewording makes sense. There are now no blanks. I think the question was fine, but if it's going to be changed, why not use something more descriptive than 'placeholder1'? Maybe 'directory1'?

File: _episodes/03-create.md [hunk 2]
```diff
@@ -207,15 +207,15 @@ draft.txt
 > 3.  When might you want to create a file this way?
 >
 > > ## Solution
-> > 1.  The touch command generates a new file called 'my_file.txt' in
+> > 1.  The touch command generates a new file called `my_file.txt` in
 > >     your home directory.  If you are in your home directory, you
-> >     can observe this newly generated file by typing 'ls' at the 
-> >     command line prompt.  'my_file.txt' can also be viewed in your
+> >     can observe this newly generated file by typing `ls` at the 
+> >     command line prompt.  `my_file.txt` can also be viewed in your
```
Comment (gcapes) [inline _episodes/03-create.md]: Is this inconsistent with #778? The name of a file is not a runnable command. Same issue again below.

Comment (rgaiacs) [reply _episodes/03-create.md]: @gcapes I created https://github.com/swcarpentry/lesson-example/pull/201 to try to have some ground rules. Could you let me know what do you thing?

File: _episodes/03-create.md (hunk 1 above)
Comment (rgaiacs) [reply _episodes/03-create.md]: a4978c30d37db478001dbfafd94c7bdd2837811d reverts to the use of underline.

**Comment by gcapes** [comment]
Thanks @rgaiacs - these changes have since been made in a0232a22 and #884 (pending merge).

**Pull request closed without merging**